### PR TITLE
Add destinationWebCategories to Application Rule in firewall Policy

### PR DESCRIPTION
**Pull request by ssripadham**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json [hunk 1]
```diff
@@ -838,7 +847,14 @@
           "items": {
             "type": "string"
           }
-        }
+        },
+        "destinationWebCategories": {
```
Comment (markcowl) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json:851]: We don't allow changes to existing api-versions in the master branch.  Did you intend to add this to a new api-version?

**Review by markcowl — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json (hunk 1 above)
Comment (markcowl) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json:851]: Also, you will need to resolve the prettier issue below by running prettier locally

Comment (ssripadham) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json]: image
Yes. this is completed

File: specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2020-06-01"
+    "version": "2020-08-01"
```
Comment (markcowl) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json:6]: you cannot change the api-version in this spot - you need to create a new api-version altogether.  Please coordinate with Mikhail on what this version should be and where it is appropriate to work on it.

**Review by markcowl — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@ssripadham Closing, as I believe you have recreated this PR against the network branch, please LMK if this needs to be reopened

**Pull request closed without merging**